routes/add_treatment/add_treatment.njk

```diff
@@ -14,8 +14,16 @@
             {{ textInput('treatment_start_date', 'treatments.form.start_date.text', { required: true, hint: 'treatments.form.start_date.hint', "autocomplete": "off"})}}
             {{ textInput('treatment_end_date', 'treatments.form.end_date.text', {required: true, hint: 'treatments.form.end_date.hint', "autocomplete": "off"})}}
             
-            {{ propCheckBoxes('treatment_treated_condition', 
-                data.conditions,"name_of_condition",{},
+
+            {% if oneValue %}
+                {% set selectedValues = ["0"] %}
+            {% else %}
+                {% set selectedValues = data.treatment_treated_condition %}
+            {% endif %}
+
+            {{ rawCheckboxes('treatment_treated_condition', 
```

Comment:
Running locally, got this error when landing on the _Add Treatment_ page with 1 condition added.

<img width="1256" alt="image" src="https://user-images.githubusercontent.com/950486/69771847-2abe4f00-115c-11ea-9389-776547bd4ad0.png">

Reply:
This has been fixed in my last push, was a capitalization error, should have been `rawCheckBoxes`.